cli/internal/taskhash/taskhash.go

```diff
@@ -133,7 +134,7 @@ func manuallyHashPackage(pkg *fs.PackageJSON, inputs []string, rootPath fs.Absol
 				if err != nil {
 					return fmt.Errorf("could not hash file %v. \n%w", name, err)
 				}
-				hashObject[strings.TrimPrefix(name, toTrim)] = hash
+				hashObject[turbopath.RelativeUnixPath(strings.TrimPrefix(name, toTrim))] = hash
```

Comment:
I actually did check the upstream inputs of this, but since it isn't type-constrained in the upstream sources that could change out from underneath me.

It's right that this makes you nervous; I should indeed force this one in particular through a checked path. (The boundary isn't library code that we would expect to maintain a consistent interface.)